shaders/src/common_instancing.glsl

```diff
@@ -20,16 +21,26 @@ void initObjectUniforms(out PerRenderableData p) {
     p.flagsChannels = objectUniforms.data[instance_index].flagsChannels;
     p.objectId = objectUniforms.data[instance_index].objectId;
     p.userData = objectUniforms.data[instance_index].userData;
+#   else
+    p.worldFromModelMatrix = objectUniforms.data[0].worldFromModelMatrix;
```

Comment:
In this case could we do `p = objectUniforms.data[0]`?

Reply:
no because that's not supported in ES2!!!